README.md

```diff
@@ -26,21 +26,48 @@ npm i @signal-noise/use-api
 
 ## Usage
 
-Here is an example of an api call to retrieve a list of people which polls every 10 seconds, with a manual refresh button.
+Here is an example of a `GET` api call to retrieve a list of people which polls every 10 seconds, with a manual refresh button.
 
 ```JSX
 import React from 'react';
 import useApi from '@signal-noise/use-api';
 import PeopleList from './PeopleList';
 
-const App = () = {
+const PeopleList = () = {
   const { data, loading, error, refresh } = useApi("https://some-api.com", 10000);
 
+  const people = data.people || [];
+
+  return (
+    <>
+      {loading && <p>Loading...</p>}
+      {error && <p>{error}</p>}
+      <button onClick={refresh} disabled={loading}>Refresh</button>
+      <PeopleList people={people} />
+    </>
+  );
+}
+```
+
+You can optionally pass in data and specify the request type. Below is a minimal example of a user search UI. (You may wish to debounce the user input 🤷‍)
+
+```JSX
+import React, { useState } from 'react';
+import useApi from '@signal-noise/use-api';
+import PeopleList from './PeopleList';
+
+const PeopleSearch = () = {
+  const [keywords, setKeywords] = useState("kazumi");
+
+  const { data } = useApi("https://some-api.com", 0, { keywords }, "post");
```

Comment:
sorry to be picky, but I find this format of method specification pretty unusual - even in the docs you capitalise the method, but then use lower case when specifying it? It wouldn't be a format I'd guess.

If it were me I'd expose constants for the methods, and also allow both UC and LC strings as a convenience.

Reply:
@marcelkornblum I have made an update that does `toLowerCase` on the `method`. I think adding constants may be overkill because:

1. Hooks should be super simple to use. I only want to import one thing for something simple like this to work.
2. We are using `module.exports` in this package to keep it compatible with how NPM likes to do things. If we wanted to export more things then we cannot have a default export. E.g.

this line...
```javascript
import useApi from "@signal-noise/use-api";
```

would have to become...
```javascript
import { useApi } from "@signal-noise/use-api";
```

I haven't really seen any published hooks without a default export.

Thoughts?

Reply:
@marcelkornblum was this cool to merge in after our discussion?